test/Saunter.Tests/AttributeProvider/DocumentGenerationTests/ClassAttributesTests.cs

```diff
@@ -37,26 +33,23 @@ public void GetDocument_GeneratesDocumentWithMultipleMessagesPerChannel(Type typ
             subscribe.OperationId.ShouldBe("TenantMessageConsumer");
             subscribe.Summary.ShouldBe("Subscribe to domains events about tenants.");
 
-            var messages = subscribe.Message.ShouldBeOfType<Messages>();
-            messages.OneOf.Count.ShouldBe(3);
+            subscribe.Message.Count.ShouldBe(3);
 
-            messages.OneOf.OfType<MessageReference>().ShouldContain(m => m.Id == "tenantCreated");
-            messages.OneOf.OfType<MessageReference>().ShouldContain(m => m.Id == "tenantUpdated");
-            messages.OneOf.OfType<MessageReference>().ShouldContain(m => m.Id == "tenantRemoved");
+            subscribe.Message.ShouldContain(m => m.MessageId == "tenantUpdated");
```

Comment:
these were originally reference checks. should we keep those around to see that referenced stuff works?
Ultimately the writer can inline the references in the final document if needed.

Also the reader can inline them during reading, if that makes things easier for testing purposes.,